inlong-sort/sort-flink/sort-flink-v1.15/sort-connectors/starrocks/src/main/java/org/apache/inlong/sort/starrocks/table/sink/table/StarRocksDynamicSinkFunctionV2.java

```diff
@@ -346,6 +340,10 @@ public void initializeState(FunctionInitializationContext functionInitialization
 
     @Override
     public void notifyCheckpointComplete(long checkpointId) {
+        if (checkpointId != Long.MAX_VALUE) {
```

Comment:
Is this check needed?

Reply:
I extract a single method for notifyCheckpointComplete(Long.MAX_VALUE). thx ~